### PR TITLE
Add support for garage gate

### PR DESCRIPTION
**Pull request by kopasiak**
I own Krispol Starcus garage gate drive with a "Gate Pro Connector" which turnes out to be compatible with this library after a few small changes.

Tested this on the 2 devices that I own and works without any issues. I'm able to open, close and set position of the gate.

Example status:
<MotionBlind mac: c8c9a305eafb, type: RollerBlind, status: Stopped, position: 100 %, angle: 0.0, limit: BothLimitsDetected, battery: None, None %, None V, charging: None, RSSI: -53 dBm, com: None>

### CODE REVIEW

File: motionblinds/motion_blinds.py [hunk 1]
```diff
@@ -31,9 +31,11 @@
 
 DEVICE_TYPE_WIFI_CURTAIN = "22000000"  # Curtain direct WiFi
 DEVICE_TYPE_WIFI_BLIND = "22000002"  # Standard Blind direct WiFi
+DEVICE_TYPE_WIFI_GATE = "22000005"  # Standard Garage gate (for example Krispol)
```
Comment (starkillerOG) [inline motionblinds/motion_blinds.py:34]:
```suggestion
DEVICE_TYPE_WIFI_GATE = "22000005"  # Garage gate direct WiFi (for example Krispol)
```

**Comment by starkillerOG** [comment]
@kopasiak Thank you for this PR, good to hear it also works with Krispol!

**Comment by krismarc** [comment]
@kopasiak I've got the same garage gate with the same connector. It works with krispol mobile app. How do I get api key in this case? I wasn't successful to connect using native motionblinds app because wifi name do not starts with 'MOTION'.

**Comment by starkillerOG** [comment]
You need the krispol app, always use the app of the brand from the blinds/gate.
Not sure where it is in the app, but probably also on the about page tap a few times.

**Comment by krismarc** [comment]
@starkillerOG there's nothing :( actually I tried with the api key given in example. I can see the device, however, I can't control it due to auth token error.

>>> list(m.device_list.values())
[<MotionBlind mac: a848facbb7c5, type: None, status: None, position: None %, angle: None, limit: None, battery: None, None %, None V, charging: None, RSSI: None dBm, com: None>]

>>> blind_1.Update()
Received actionResult: 'AccessToken error', when sending message: '{'msgType': 'WriteDevice', 'mac': 'a848facbb7c5', 'deviceType': '22000005', 'AccessToken': 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx', 'msgID': '20241130111142521', 'data': {'operation': 5}}', got response: '{'msgType': 'WriteDeviceAck', 'mac': 'a848facbb7c5', 'deviceType': '22000005', 'msgID': '20241130111142521', 'data': {'currentPosition': 100, 'currentAngle': 0, 'RSSI': -73}, 'actionResult': 'AccessToken error'}'

**Comment by kopasiak** [comment]
Go to the krispol app, open settings and navigate to About page then
quickly tap krispol logo a few times and you will get a popup with API key

**Comment by krismarc** [comment]
Ok, I just  found it a moment ago. Thank you!
For other people. You need to press on your account avatar so it opens user account preferences and there's 'about' :)

**Comment by krismarc** [comment]
Thanks guys! Worked :)

**Pull request merged**